src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -159,7 +159,7 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
                             "${qnIdx}", Integer.toString(qnIdx),
                             "${responseIdx}", Integer.toString(responseIdx),
                             "${optionIdx}", "0",
-                            "${disabled}", sessionIsOpen ? "" : "disabled=\"disabled\"",
+                            "${disabled}", sessionIsOpen ? "" : "disabled=\"\"",
```

Comment:
I think we should simply have `disabled` and not `disabled=""`. I am not sure though...